packages/injector/loadDependency/index.js

```diff
@@ -9,13 +9,28 @@ import {
 import extractDependencies from './extractDependencies'
 import compose from '../compose'
 import circularDependencyCheck from './circularDependencyCheck'
+import builtIns from 'builtin-modules'
+import es6Require from '@b-mo/es6-require'
+const isPackageDependency = ({ dependency, pkg = {}}) => {
+  return has(pkg, `dependencies.${dependency}`)
+}
 
 const loadDependency = async (manifest, name, dependency, dependencies, circleChecked, dependencyProperty) => {
   const dependencyPath = `${dependencyProperty}.${name}`
   if (has(manifest, dependencyPath)) {
     return manifest
   }
 
+  if (builtIns.indexOf(name) !== -1) {
```

Comment:
If you move this if conditional into a function, then you can add this new function as an OR condition to line 29 (since the logic is the same on lines 25-26 and 30-31). It would be a bit easier to read that way (though about the same number of code lines).

Reply:
done